content/cn/template.json

```diff
@@ -1,28 +1,41 @@
 {
-  "browser-title":"io.js - JavaScript I/O",
-  "logo-text":"io.js",
-  "heading-languages":"其他语言",
-  "faq-link":"常见问题",
-  "es6-link":"ES6",
-  "api-link":"API 文档",
-  "issues-link":"GitHub Issues",
-  "org-link":"GitHub Org",
-  "irc-link":"IRC Chat",
-  "irc-logs-link":"Logs",
-  "gov-link":"项目管理",
+  "browser-title": "io.js - JavaScript I/O",
+  "contribute-message": "See something you like? Want to help? Visit https://github.com/iojs/iojs-cn to contribute",
+  "logo-text": "io.js",
+  "heading-languages": "其他语言",
+  "faq-link": "常见问题",
+  "es6-link": "ES6",
+  "api-link": "API 文档",
+  "issues-link": "GitHub Issues",
+  "org-link": "GitHub Org",
+  "irc-link": "IRC Chat",
+  "irc-logs-link": "Logs",
+  "gov-link": "项目管理",
   "verbose_version": "{{project.current_version}} 版本",
   "downloads": {
+    "linux": "Linux",
+    "win32": "Win32",
+    "win64": "Win64",
+    "mac": "Mac",
     "all": "其他"
   },
   "home": {
     "download_links": "下载 {{> current_download_links}} 版本。",
+    "custom_changelog": "{{link '更新日志' 'https://github.com/iojs/iojs-cn/blob/gh-pages/CHANGELOG.md'}}",
+    "faq_verbose": "{{link 'pages.faq_verbose'}}",
     "nightly_releases": "{{link '每日构建版本' 'https://iojs.org/download/nightly/'}} 可用于测试。",
     "short_description": "{{link 'website'}} 是一个衍生自 {{link 'nodejs'}}，并兼容 {{link 'npm'}} 的开发平台。",
-    "slogan": "将 {{link 'pages.es6'}} 带入 Node 社区！"
+    "slogan": "将 {{link 'pages.es6'}} 带入 Node 社区！",
+    "news_link": "{{link '每周播报' 'http://cn.iojs.org/weekly/'}} <small>核心特性和社区动态 ({{link 'Medium' 'https://medium.com/node-js-javascript/'}})</small>"
   },
   "links": {
+    "nodejs": "Node.js™",
+    "npm": "npm",
+    "website": "io.js",
     "pages": {
-      "changelog": "更新日志",
```

Comment:
Did you still want this line fully removed? https://github.com/iojs/website/blob/master/source/templates/releases.html#L21 also refers to the i18n text which feeds the string here: https://iojs.org/en/releases.html

FYI I haven't figured out how to make those per-version links i18n friendly yet